en/05_Contributing/17_Managing_Security_Issues.md

```diff
@@ -0,0 +1,113 @@
+
+This process is relevant when a potential vulnerability is reported confidentially and the Silverstripe CMS development team is in a position to prepare a patch prior to the public disclosure of the vulnerability.
+
+This process is usually started once someone [reports a security issue](Issues-and-Bugs/##security-issue).
+
+### When receiving a report
+
+* An automated response is sent back to the reporter to acknowledge receipt of their vulnerability report.
+* Perform an initial assessment of the report.
+* [Create a issue in our private security repository](https://github.com/silverstripe-security/security-issues/issues/new) unless to report is obviously invalid. e.g. SPAM.
+* If encrypted information is provided, attach it to the private security issue.
+* Reply to [security@silverstripe.org](mailto:security@silverstripe.org) ONLY and provide a link to the private security issue. Keep most of the discussion on GitHub.
+* Perform initial criticality assessment. Validate assessment with at least one other member of the security team before replying to the reporter with your conclusion. Ensure the reporter is given a justification for all issues we classify or demote as non-security vulnerabilities. You may need to seek additional information from the reporter before completing the criticality assessment.
+* Add the new issue in the "Backlog" on the [project board](https://github.com/silverstripe-security/security-issues/projects/1).
+* Use the [CVSS Calculator](https://nvd.nist.gov/vuln-metrics/cvss/v3-calculator) to determine the issue severity.
+* Once the issue is confirmed, create a _draft security advisory_ against the relevant GitHub repository. This will allow you to request a CVE.
+* Once a CVE has been assigned, respond to issue reporter and add it to the Github issue.
+* Ask the reporter if they want to be credited for the disclosure and under what name.
+* Clarify who picks up and owns the issue (assign in Github).
+  The owner can be separate from the developer resolving the issue,
+  their primary responsibility is to ensure the issue keeps moving through the process correctly.
+
+### When developing a fix
+
+* Ensure you're working on the oldest supported minor release branch of every supported major release (see [Supported Versions](#supported-versions))
+* Move the issue into "In Progress" on the [project board](https://github.com/silverstripe-security/security-issues/projects/1)
+* Add fixes on the [http://github.com/silverstripe-security](http://github.com/silverstripe-security) repo. Don't forget to update branches from the upstream repo.
+* Ensure that all security commit messages are prefixed with the CVE. E.g. "[CVE-2019-001] Fixed invalid XSS"
+
+### Before release
+
+* For issues rated "high" or "critical" (CVSS of >=7.0), post a pre-announcement to the [security pre-announcement list](https://groups.google.com/a/silverstripe.com/forum/#!forum/security-preannounce).
```

Comment:
That URL gives me a 403 error

Reply:
The link is restricted to very small number of people. There's probably no point having it here.

Reply:
I agree, should probably be removed if not everyone who can manage the security process has access to do this step.

Reply:
There's lots of things on this page that most people won't have access to. I'm thinking we probably need to grant the entire CMS squad access to this mailing list

Reply:
That would make sense. Anyone who is part of the "Silverstripe CMS security team" should probably have whatever access is required to do all of the steps in this document.

Reply:
Added the entire Squad to the list.